pgxn/neon/control_plane_connector.c

```diff
@@ -222,6 +224,104 @@ ErrorWriteCallback(char *ptr, size_t size, size_t nmemb, void *userdata)
 	return nmemb;
 }
 
+
+static size_t
+ResponseWriteCallback(char *ptr, size_t size, size_t nmemb, void *userdata)
+{
+	appendBinaryStringInfo((StringInfo)userdata, ptr, size*nmemb);
+	return nmemb;
+}
+
+void
+RequestShardMapFromControlPlane(ShardMap* shard_map)
```

Comment:
do we really need to make this request? Can it come from the control plane in spec?

Reply:
There's a slack thread on this: https://neondb.slack.com/archives/C05EPKK8K6G/p1699545994695979?thread_ts=1699541077.613409&cid=C05EPKK8K6G

Reply:
> do we really need to make this request? 

No, do not need.
I removed this changes in control_plane_connector.c and extract information about shards connection n string from "neon.pageserver_connstring" (assuming that them are Emma separated).

Reply:
This changes are reverted